cclib/bridge/cclib2horton.py

```diff
@@ -137,17 +183,18 @@ def makecclib(iodat):
             attributes["mocoeffs"] = [iodat.mo.coeffs[: iodat.mo.norba].T]
             if iodat.mo.kind == "unrestricted":
                 attributes["mocoeffs"].append(iodat.mo.coeffs[iodat.mo.norba :].T)
-        if hasattr(iodat, "spinpol"):
+        if hasattr(iodat, "spinpol") and isinstance(iodat.spinpol, int):
             # IOData stores 2S, ccData stores 2S+1.
             attributes["mult"] = iodat.spinpol + 1
         if hasattr(iodat, "atnums"):
+            attributes["atnums"] = numpy.asanyarray(iodat.atnums)
+        if hasattr(iodat, "atcorenums") and isinstance(iodat.atnums, numpy.ndarray):
             # cclib stores num of electrons screened out by pseudopotential
```

Comment:
These comments were placed in a wrong (a block above) location in the previous PR. This should fix it.